README.md

```diff
@@ -18,24 +18,25 @@ on `sourceror`.
 > mix recode.help
 
 Design tasks:
-TagFIXME          # Checker   - Checks if there are FIXME tags in the sources.
-TagTODO           # Checker   - Checks if there are TODO tags in the sources.
-Readability tasks:
-AliasExpansion    # Corrector - Exapnds multi aliases to separate aliases.
-AliasOrder        # Corrector - Checks if aliases are sorted alphabetically.
-EnforceLineLength # Corrector - Forces expressions to one line.
-Format            # Corrector - Does the same as `mix format`.
-PipeFunOne        # Corrector - Add parentheses to one-arity functions.
-SinglePipe        # Corrector - Pipes should only be used when piping data through multiple calls.
-Specs             # Checker   - Checks for specs.
+TagFIXME            # Checker   - Checks if there are FIXME tags in the sources.
+TagTODO             # Checker   - Checks if there are TODO tags in the sources.
+Readability tasks:  
+AliasExpansion      # Corrector - Exapnds multi aliases to separate aliases.
+AliasOrder          # Corrector - Checks if aliases are sorted alphabetically.
+EnforceLineLength   # Corrector - Forces expressions to one line.
+Format              # Corrector - Does the same as `mix format`.
+PipeFunOne          # Corrector - Add parentheses to one-arity functions.
+SinglePipe          # Corrector - Pipes should only be used when piping data through multiple calls.
+Specs               # Checker   - Checks for specs.
 Refactor tasks:
-FilterCount       # Corrector - Checks calls like Enum.filter(...) |> Enum.count().
-Nesting           # Checker   - Checks code nesting depth in functions and macros.
+FilterCount         # Corrector - Checks calls like Enum.filter(...) |> Enum.count().
+Nesting             # Checker   - Checks code nesting depth in functions and macros.
+UnnecessaryIfUnless # Corrector - Changes `if expr, do: true, else: false` to `expr`
 Warning tasks:
-Dbg               # Corrector - There should be no calls to dbg.
-IOInspect         # Corrector - There should be no calls to IO.inspect.
-TestFileExt       # Corrector - Checks the file extension of test files.
-UnusedVariable    # Corrector - Checks if unused variables occur.
+Dbg                 # Corrector - There should be no calls to dbg.
+IOInspect           # Corrector - There should be no calls to IO.inspect.
+TestFileExt         # Corrector - Checks the file extension of test files.
+UnusedVariable      # Corrector - Checks if unused variables occur.
```

Comment:
I feel a bit off about all these whitespace changes.  If you'd prefer a column cut-off with overflow I'm happy to undo.

Reply:
The whitespace changes are ok.